src/utils/Utils.cpp

```diff
@@ -430,4 +432,18 @@ for x in range(256):
 			NRF_LOG_INFO("Done waiting");
 		#endif
 	}
+
+    void leaveValidation()
+    {
+        //ret_code_t err = sd_softdevice_disable();
+        //APP_ERROR_CHECK(err);
+
+        nrf_nvmc_write_byte(0x10001080, 0xFC);
```

Comment:
Please use SDK constant &NRF_UICR->CUSTOMER[0] instead of 0x10001080
(and in other places where this value is hardcoded)